packages/frontend/package.json

```diff
@@ -21,8 +21,10 @@
     "scripts": {
         "lint": "eslint --ext .js,.ts,.tsx ./ && prettier --check .",
         "format": "eslint --fix --ext .tsx,.ts,.js ./ && prettier --write .",
-        "build": "node ./scripts/build.js",
-        "build:prod": "NODE_ENV=production node ./scripts/build.js",
+        "build": "yarn build:dev",
+        "build:production": "NODE_ENV=production ENVIRONMENT=prod node ./scripts/build.js",
+        "build:staging": "ENVIRONMENT=staging node ./scripts/build.js",
```

Comment:
Maybe we should build production code also for staging as it's meant to be an as-close-as-possible environment to production.

Reply:
Fixed in 9e02924b0b2b9b66b7c58af79aee469fbaceca85.